enterprise/frontend/src/embedding-sdk-bundle/components/private/SdkBreadcrumbs/SdkBreadcrumbs.unit.spec.tsx

```diff
@@ -111,7 +111,7 @@ describe("SdkBreadcrumbs", () => {
     });
 
     await waitFor(() => {
-      expect(screen.getByText("Last edited by")).toBeInTheDocument();
+      expect(screen.getByText("This collection is empty")).toBeInTheDocument();
```

Comment:
When I added in the `setupCollectionByIdEndpoint` mock endpoint, it changed the message that is shown. I guess previously there was no response from the api and it defaulted to 'Last edited by'